src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,1010 @@
+| **AUDIO**    | Audio signal                                                                                                                    |
+| **PD**       | Photodiode                                                                                                                      |
+| **EYEGAZE**  | Eye Tracker gaze                                                                                                                |
+| **PUPIL**    | Eye Tracker pupil diameter                                                                                                      |
+| **BEH**      | Behavioral signals                                                                                                              |
+| **MISC**     | Miscellaneous                                                                                                                   |
+| **SYSCLOCK** | System time showing elapsed time since trial started                                                                            |
+| **ADC**      | Analog to Digital input                                                                                                         |
+| **DAC**      | Digital to Analog output                                                                                                        |
+| **REF**      | Reference channel                                                                                                               |
+| **OTHER**    | Any other type of channel                                                                                                       |
+
+## Electrodes description (`*_electrodes.tsv`)
+
+Electrodes are the physical recording sites that make electrical contact with neural tissue to capture electrophysiological signals.
+
+The electrode positions and properties are stored in a `.tsv` file (amplifier information is in `channels.tsv`).
+
+This file contains the following information:
+
+-   The electrode name
+-   The electrode coordinates in 3 columns (`xyz`) (use `n/a` for values if a dimension is absent). By default, this is the position on the probe (not the brain).
+-   The ID of the probe the electrode is located on
+
+{{ MACROS___make_columns_table("microephys.microephysElectrodes") }}
+
+### Example `*_electrodes.tsv`
+
+**Extracellular electrophysiology example (probe-relative coordinates):**
+
+```tsv
+name	probe_name	hemisphere	x	y	z	impedance	shank_id	size	material	location
+e001	probe01	L	0	0	0	1.2	0	15	iridium-oxide	MOp
+e002	probe01	L	0	0	25	1.1	0	15	iridium-oxide	MOp
+e003	probe01	L	0	0	50	1.3	0	15	iridium-oxide	MOp
+e004	probe01	L	0	0	75	1.4	0	15	iridium-oxide	MOp
+e005	probe02	R	0	0	0	2.1	n/a	12	tungsten	CA1
+e006	probe02	R	0	0	15	2.3	n/a	12	tungsten	CA1
+e007	probe02	R	0	0	30	1.9	n/a	12	tungsten	CA1
+e008	probe02	R	0	0	45	2.0	n/a	12	tungsten	CA1
```

Comment:
```suggestion
e001	probe01	L	0	0	n/a	1.2	0	15	iridium-oxide	MOp
e002	probe01	L	0	25	n/a	1.1	0	15	iridium-oxide	MOp
e003	probe01	L	0	50	n/a	1.3	0	15	iridium-oxide	MOp
e004	probe01	L	0	75	n/a	1.4	0	15	iridium-oxide	MOp
e005	probe02	R	0	0	n/a	2.1	n/a	12	tungsten	CA1
e006	probe02	R	0	15	n/a	2.3	n/a	12	tungsten	CA1
e007	probe02	R	0	30	n/a	1.9	n/a	12	tungsten	CA1
e008	probe02	R	0	45	n/a	2.0	n/a	12	tungsten	CA1
```

Reply:
I would also be fine with moving position to x and having both y and z be n/a, but I think what I have proposed is the more common convention